pybaseball/utils.py

```diff
@@ -64,8 +64,19 @@ def validate_datestring(date_text: Optional[str]) -> date:
     except (AssertionError, ValueError):
         raise ValueError("Incorrect data format, should be YYYY-MM-DD")
 
+def most_recent_season() -> int:
```

Comment:
you're calling this function in a lot of different places, how about adding an `functools.lru_cache` decorator?